eras/alonzo/impl/testlib/Test/Cardano/Ledger/Alonzo/TreeDiff.hs

```diff
@@ -172,3 +176,17 @@ instance
         , ("pwcExUnits", toExpr pwcExUnits)
         , ("pwcCostModel", toExpr pwcCostModel)
         ]
+
+deriving instance Generic (TransactionScriptFailure era)
```

Comment:
Why is this a standalone instance instead of placed right next to the type definition?

Reply:
It's needed only for tests so I didn't think it was necessary to add it to the non-test module where the type is defined.

Reply:
However, I've taken it out now, because it's not needed as you pointed out